pkg/api/v1alpha1/tinkerbelldatacenterconfig.go

```diff
@@ -74,6 +74,18 @@ func validateDatacenterConfig(config *TinkerbellDatacenterConfig) error {
 		return fmt.Errorf("TinkerbellDatacenterConfig: invalid tinkerbell ip: %v", err)
 	}
 
+	if config.Spec.IsoBoot {
+		if config.Spec.HookIsoURL != "" {
+			if _, err := url.ParseRequestURI(config.Spec.HookIsoURL); err != nil {
+				return fmt.Errorf("parsing isoURL: %v", err)
```

Comment:
nothing to hold up the PR, but something to think about. Because our code base doesn't have a good presentation layer, error messages like this most of the time get surfaced to the user. So while normally an error message that references some internal or local field or variable name is ok, you should think about what name makes sense from the user's perspective.